src/shadowbox/README.md

```diff
@@ -119,15 +119,15 @@ Remove an access key
 curl --insecure -X DELETE $API_URL/access-keys/2
 ```
 
-Set an access key data limit
-(e.g. limit outbound data transfer for access key 2 to 1MB over 30 days)
+Set a data limit for all access keys
+(e.g. limit outbound data transfer access keys to 1MB over 30 days)
 ```
-curl -v --insecure -X PUT -H "Content-Type: application/json" -d '{"limit": {"bytes": 1000}}' $API_URL/access-keys/2/data-limit
+curl -v --insecure -X PUT -H "Content-Type: application/json" -d '{"limit": {"bytes": 1000}}' $API_URL/experimental/access-key-data-limit
 ```
 
-Remove an access key data limit
+Remove the access key data limit
 ```
-curl -v --insecure -X DELETE $API_URL/access-keys/2/data-limit
+curl -v --insecure -X DELETE $API_URL/experimental/access-key-data-limit
```

Comment:
My worry with `server-data-limit` is that it can be interpreted as the total data limit for the server, as opposed to "the data limit applied to individual keys". If we decide to make the experiment a feature, I think the this endpoint should be `server/access-key-data-limit`, which would address your point.